release/models/pcep/openconfig-pcep.yang

```diff
@@ -0,0 +1,377 @@
+          independently of each other(RFC4655).
+
+        - Passive Stateful PCE:  a PCE that uses LSP state information
+          learned from PCCs to optimize path computations, but it does
+          not actively update LSP state(RFC8051). - Active Stateful
+          PCE:  a PCE that may issue recommendations to the network by
+          using the Delegation mechanism to update LSP parameters in
+          those PCCs that delegate control over their LSPs to the PCE
+          (RFC8051).";
+    reference
+      "RFC8051: Applicability of a Stateful Path Computation Element (PCE)";
+  }
+
+  typedef lsp-control-type {
+    type enumeration {
+      enum PCE_DELEGATED {
+        description
+          "PCC(Router) delegates control over LSP path to PCE (stateful active)";
+      }
+      enum PCC_CONTROLLED {
+        description
+          "PCC(Router) don't allow or revoke control over LSP path to
+          PCE (stateless)";
+      }
+      enum PCC_REPORT_ONLY {
+        description
+          "PCC(Router) only inform to a PCE whenever the state of an LSP changes,
+          but without delegating control (stateful passive)";
+      }
+    }
+    description
+      "The LSP path can be delegated to PCE, locally controled by PCC
+      or only report information state to the PCE.";
+    reference
+      "RFC8231: Path Computation Element Communication Protocol (PCEP)";
+  }
+
+  grouping pcep-servers-top {
+    description
+      "Configuration and state data for establishing a PCEP session with pce servers.";
+
+    container path-computation-servers {
+      description
+        "Defines pce server parameters.";
+	
+      list path-computation-server {
+        key "pce-server-address";
+	
+        description
+          "Configuration and state information for communication with a pce server.";
+
+        leaf pce-server-address {
+          type leafref {
+            path "../config/pce-server-address";
+          }
+          description
+            "IPv4 address of pce server.";
+        }
+
+        container config {
+          description
+            "PCE server configuration.";
+			
+          uses pce-server-config;
+        }
+		
+        container state {
+          config false;
+          description
+            "PCE server state.";
+			
+          uses pce-server-config;
+        }
+		
+	uses pcep-timers-top;
+	uses pcep-authentication-top;
+      }
+    }
+  }
+	
+  grouping pce-server-config {
+    description
+      "Configuration data for establishing a PCEP session with pce servers.";
+
+    leaf pce-server-address {
+      type oc-inet:ip-address;
+      description
+      	"The destination IP address of the PCE server for PCEP service.
+      	Repeat this step to specify multiple PCE servers to provide redundancy.
+      	When multiple candidate servers are specified for a single client,
+      	they are prioritized according to preference.";
+    }
+	
+    leaf pce-server-id {
```

Comment:
Could this just be `id`, since the list is already called `path-computation-server`?